extensions/plugins/system/redcore/redcore.php

```diff
@@ -316,6 +316,23 @@ public function onBeforeCompileHead()
 				unset($doc->_scripts[JURI::root(true) . '/templates/' . $template . '/js/jui/bootstrap.js']);
 				unset($doc->_scripts[JURI::root(true) . '/templates/' . $template . '/js/jui/bootstrap.min.js']);
 			}
+
+			// Remove permission.js. Specially for Joomla 3.6 and make sure run only on redCORE's base extensions
+			if ($isAdmin && version_compare(JVERSION, '3.6.0', '>='))
+			{
+				$redcoreExtensions = RComponentHelper::getRedcoreComponents();
+				$redcoreExtensions[] = 'com_redcore';
+				$option = JFactory::getApplication()->input->getCmd('option');
+
+				if (!empty($option) && in_array($option, $redcoreExtensions)
```

Comment:
this is not ok, it will still go through... you need to put it into brackets...
if (!empty($option) && in_array($option, $redcoreExtensions)
&& (isset($doc->_scripts[JURI::root(true) . '/media/system/js/permissions.js'])
|| isset($doc->_scripts[JURI::root(true) . '/media/system/js/permissions-uncompressed.js'])))
but I dont think you need that 2 checks... or make them in a separate if below...